components/tidb_query/src/codec/row_codec.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use crate::codec::{table, Datum, Result};
+
+pub mod v_128;
```

Comment:
why name 128?

Reply:
since the version code is 128 😅.  I'll rename it to `v_2` later, as following file structure. 
Will this change make it less confusing?
 
```
|-- row_codec.rs
|-- row_codec
     |-- v1.rs
     |-- v1
          |-- encoder.rs
          |-- decoder.rs
     |-- v2.rs
     |-- v2
          |-- encoder.rs
          |-- decoder.rs
```

Reply:
As @breeswish suggesting in https://github.com/tikv/tikv/pull/5497#discussion_r326904756, prefer to move the original codec logic to `codec::row::v1`.